TidepoolOnboardingKitUI/View Models/OnboardingViewModel.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  OnboardingViewModel.swift
+//  TidepoolOnboardingKitUI
+//
+//  Created by Darin Krauss on 1/29/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+import Foundation
+import LoopKit
+import LoopKitUI
+
+class OnboardingViewModel: ObservableObject, OnboardingNotifying, CGMManagerCreateNotifying, CGMManagerOnboardNotifying, PumpManagerCreateNotifying, PumpManagerOnboardNotifying, ServiceCreateNotifying, ServiceOnboardNotifying, CompletionNotifying {
+    weak var onboardingDelegate: OnboardingDelegate?
+    weak var cgmManagerCreateDelegate: CGMManagerCreateDelegate?
+    weak var cgmManagerOnboardDelegate: CGMManagerOnboardDelegate?
+    weak var pumpManagerCreateDelegate: PumpManagerCreateDelegate?
+    weak var pumpManagerOnboardDelegate: PumpManagerOnboardDelegate?
+    weak var serviceCreateDelegate: ServiceCreateDelegate?
+    weak var serviceOnboardDelegate: ServiceOnboardDelegate?
+    weak var completionDelegate: CompletionDelegate?
+
+    private let cgmManagerProvider: CGMManagerProvider
+    private let pumpManagerProvider: PumpManagerProvider
+    private let serviceProvider: ServiceProvider
+
+    @Published var isWelcomeComplete: Bool = false
+
+    init(cgmManagerProvider: CGMManagerProvider, pumpManagerProvider: PumpManagerProvider, serviceProvider: ServiceProvider) {
+        self.cgmManagerProvider = cgmManagerProvider
+        self.pumpManagerProvider = pumpManagerProvider
+        self.serviceProvider = serviceProvider
+    }
+
+    private func notifyComplete() {
+        self.completionDelegate?.completionNotifyingDidComplete(self)
+    }
+}
+
+extension OnboardingViewModel: OnboardingDelegate {
+    func onboardingNotifying(hasNewTherapySettings therapySettings: TherapySettings) {
+        onboardingDelegate?.onboardingNotifying(hasNewTherapySettings: therapySettings)
+    }
+}
+
+extension OnboardingViewModel: CompletionDelegate {
+    func completionNotifyingDidComplete(_ object: CompletionNotifying) {
+        if let prescriptionReviewUICoordinator = object as? PrescriptionReviewUICoordinator {
+            prescriptionReviewUICoordinator.dismiss(animated: true)
+            notifyComplete()
```

Comment:
This if statement seems out of place. How does the `OnboardingViewModel` even know that `PrescriptionReviewUICoordinator` even exist?

Reply:
Currently the `PrescriptionReviewUICoordinator` is just "stuck on" (with programming duct tape) to the end of the onboarding workflow just so it still is available for folks to run. When I get to the later tickets of correctly integrating it into the correct location in onboarding this will fit better. Not planning on changing this now since later work will resolved this.